src/components/Dropdown/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './Dropdown';
```

Comment:
I don't remember this approach in this project. Maybe it is better to stay with default export from dropdown for consistency?